src/styles/variables.scss

```diff
@@ -0,0 +1,18 @@
+$color-dark: #222222; //semi black
+$color-light: #FAFAFA; //semi white
+$color: #749C75; //green
```

Comment:
$color.... "kolor" jaki? Główny? Tekstu? Jeśli tak, to w jakim wariancie? Warto to doprecyzować. Zwłaszcza, że wyżej masz $color-dark i $color-light, czyli inny programista mógłby założyć, że są to zmienne do koloru tekstu dla trybu jasnego i ciemnego. Jeśli tak, to do czego ma być $color?